code401challenges/src/test/java/code401challenges/LinkedListTest.java

```diff
@@ -69,4 +69,40 @@ public void emptyTest(){
                 randomList.isEmpty()
         );
     }
+    @Test
+    public void appendTest(){
+        randomList.insert(3);
+        randomList.insert(2);
+        randomList.insert(1);
+        assertEquals(
+                "This should give back true.",
+                4,
+                randomList.append(4).value
+        );
+    }
+    @Test
+    public void insertBeforeTest(){
+        randomList.insert(3);
+        randomList.insert(2);
+        randomList.insert(1);
+        randomList.insertBefore(3,4);
+        assertEquals(
+                "This should give back true.",
+                "1, 2, 4, 3, ",
+                randomList.toString()
```

Comment:
These tests using toString() make me a little nervous, just because they will probably fail if the implementation of toString() changes even though insertBefore() is still working fine.